UEBPrep/app/assets/javascripts/cards.js

```diff
@@ -0,0 +1,56 @@
+$(document).ready(function() {
+    $('#card_type').change(function(){
+        $('#card_title_tr').remove();
+        $('#card_textarea_tr').remove();
```

Comment:
Yep, can totally change if need be. I did it this way initially because I know they wanted several different card types than the 2 basic ones we had here and I didn't know if those formats would make it too complex for a simple display:none or whatever.